src/plugins/intel_cpu/src/nodes/kernels/kernel_utils.hpp

```diff
@@ -0,0 +1,547 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "cpu/x64/jit_generator.hpp"
+#include <ie_common.h>
+#include <dnnl_types.h>
+#include <set>
+
+namespace ov {
+namespace intel_cpu {
+
+template <typename Vmm>
+class vRefWrap;
+
+#define r64Ref() rRefWrap<Xbyak::Reg64>(this, r64Pool[getRegIdx()])
+
+class jitKernelBase: public dnnl::impl::cpu::x64::jit_generator {
```

Comment:
Usually in "camel" case the class is named starting from capital first letter.
Please, renamed it to `JitKernelBase` or `JITKernelBase`

Reply:
Done